Added some meta tags in examples

adisrivas opened the pull request.

adisrivas committed:
examples: add viewport and charset where missing

closes #3860

wesleytodd commented:
Hey @adisrivas, while I have nothing against this, I also don't see any value in adding it.  Things I would like to know:

1. Does this make some experience better for express users?
2. Are there other examples which also have (or dont have) this meta tag? If it is not consistent, would it be better to make them all consistent in one PR?

adisrivas commented:
Hey @wesleytodd, as these meta tags are by and large present in every template hence I added them in ejs example. Yes, there were other examples also besides ejs which do not had it, thus I have now added it in all of them.

wesleytodd commented:
Awesome on the response to number 2, but the real value was an answer to question 1.  Does this meaningfully change the experience for developers who view the examples?

dougwilson commented:
I _guess_ we can just add these, since (1) the meta charset would just make those the same as others that already have it and (2) the viewport change is just sadly the reality of a working mobile view, and these pages have basically no content, so 🤷‍♂.
